transactions/README.md

```diff
@@ -11,7 +9,7 @@ When running this module, the server has to have already replaced its journal by
 
 You can do this either by
 
-* running `./prepare-and-start-server.sh -t <path to prepared journal> -u <path to unzipped EAP>`
+* running `./prepare-and-start-server.sh -t <path to prepared journal> -u <path to unzipped EAP>` with optional `-d` when running tests in domain mode
```

Comment:
does it mean that running with `-d` means testing server in domain mode? If yes, it should be better described as currently it sounds as some option which makes sense in domain mode only without any explanation what it does.